telemetry/data_pipeline_mvp/README.md

```diff
@@ -0,0 +1,15 @@
+# software test mvp
+
+installation instructions
+pip install pyserial 
+
+check serial ports, check code, TODO: include args 
+
+- install grafana
+- install redis 
+- run makes and tests and stuff
+- install redis time series
```

Comment:
include installation instructions? maybe a `setup.sh`? Make it accessible to anyone trying to run this software